index.bs

```diff
@@ -9739,13 +9739,13 @@ Methods</h5>
 				Get</a>(O=<i>processorCtor</i>,
 				P="prototype")</code>.
 
-			<li id="check-prototype-type"> If the result of
```

Comment:
Please verify that the ids aren't used anywhere, here and below.

Otherwise looks good.

Reply:
Yes, I've checked it when I submitted the PR. These IDs were not used anywhere.